cockpit_api/src/main/java/com/cockpit/api/model/dto/JiraSprintDTO.java

```diff
@@ -0,0 +1,44 @@
+package com.cockpit.api.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Date;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JiraSprintDTO {
+    private Integer originBoardId;
+    private Date endDate;
+    private String name;
+    private Integer id;
+    private String state;
+    private Date startDate;
+    private Date completeDate;
+
+    public Integer getOriginBoardId() {
+        return originBoardId;
+    }
+
+    public Date getEndDate() {
+        return endDate;
```

Comment:
JiraSprintDto do we still use this class ?

Reply:
no, removed